README.md

```diff
@@ -35,6 +35,15 @@ Flutter support for (Neo)vim
 
 > **NOTE**: install `dart-vim-plugin` plugin if your (neo)vim detect filetype incorrect
 
+### Ubuntu snap
+
+If you installed flutter using default [snap instructions](https://flutter.dev/docs/get-started/install/linux#install-flutter-using-snapd). You should add the following configuration to `:CocConfig`
+
+```
+  "flutter.sdk.flutter-lookup": "echo $HOME/snap/flutter/common/flutter/bin/flutter",
+  "flutter.sdk.dart-lookup": "echo $HOME/snap/flutter/common/flutter/bin/dart",
```

Comment:
Can you check if only adding the flutter lookup is sufficient?

Reply:
I checked the source code, seems just adding flutter part is not sufficient

Reply:
you mean the source code of this plugin?
Because it uses the flutter lookup to find the sdk home directory which is again used to find the dart analysis server.
The dart command should simply work since it doesnt need to know the sdk location from there it just calls `dart ...` which should already work for you without the settings.
(A pr from actually also sets the dart command using the flutter lookup)

Reply:
only `flutter.sdk.flutter-lookup` settings should work, `flutter.sdk.dart-lookup` will use if can not find sdk location through flutter command or flutter-lookup. can you test with each setting only if it works? so we can specify like:

```
 "flutter.sdk.flutter-lookup": "echo $HOME/snap/flutter/common/flutter/bin/flutter"
```

Or

```
  "flutter.sdk.dart-lookup": "echo $HOME/snap/flutter/common/flutter/bin/dart",
```